.dockerignore

```diff
@@ -12,4 +12,3 @@ __pycache__/
 *$py.class
 
 .venv
-
```

Comment:
ah sorry i mean this removing of the last line here @berrydenhartog

Reply:
The last line is automatically added by the .editorconfig (line 13) unless we want to specify differently behavior for the dockerignore file. 

Is there a reason why you want the last line to be removed? if so, then i can add the config to the editorconfig

Reply:
To be completely honest, It doesn't bother me too much. But i reckon is can be annoying if we will remove this line everytime we make a new repo so then I would opt to change the editorconfig indeed

Reply:
I do not understand why you want to remove it?

Reply:
Well I would like to keep the newline at the end of the file as this is pep8 right? But you removed it here specifically. But I think the discussion is getting too big for such a small thing so we can leave it like you did